bin/aursync

```diff
@@ -53,12 +64,12 @@ download() {
     declare gzdir=$1 startdir=$PWD
     cd_safe "$gzdir"
 
-    if type -P aria2c >/dev/null 2>&1; then
+    if ((pubkeypin==0)) && type -P aria2c >/dev/null 2>&1; then
         aria2c --stderr=true -i -
     elif type -P parallel >/dev/null 2>&1; then
-        parallel --will-cite -X -j +5 --nice 10 --halt soon,fail=1 "curl -fLsS --remote-name-all {}"
+        parallel --will-cite -X -j +5 --nice 10 --halt soon,fail=1 "curl ${curl_args[*]} {}"
```

Comment:
I am unsure whether this is the correct expansion to use.
Shellcheck gave me an error when I used `[@]` here.